glide-core/src/client/mod.rs

```
@@ -525,11 +525,31 @@ impl Client {
                     Some(ResponsePolicy::AllSucceeded),
                 ));
                 let mut cmd = redis::cmd("AUTH");
+                if let Ok(Some(username)) = self.get_username().await {
```

don't ignore errors, instead, return the error to the caller by using '?':
```
if let Some(username) = self.get_username().await? {